pkg/kritis/review/review.go

```diff
@@ -159,6 +160,22 @@ func (r Reviewer) findUnsatisfiedAuths(image string, auths []v1beta1.Attestation
 	return notAttestedBy, attestedByAny
 }
 
+func (r Reviewer) hasSatisfiedAuth(image string, auths []v1beta1.AttestationAuthority, c metadata.ReadOnlyClient) bool {
```

Comment:
Could you add a comment about the purpose of this function. Otherwise LGTM

Reply:
Thanks for your review, a comment has been added.